config.toml

```diff
@@ -79,50 +79,50 @@ themesDir = "node_modules/"
   weight = 5
   
   
-[[menu.sidebar]]
-  name = "Adopters"
-  url = "/adopters/"
-  weight = 1
-  identifier = "adopters"
+# [[menu.sidebar]]
+#   name = "Adopters"
+#   url = "/adopters/"
+#   weight = 1
+#   identifier = "adopters"
```

Comment:
All these have been commented out is what I did some test and I find they are not in use. Not sure if m missing anything and why they are here.

Reply:
The page is configured to hide the sidebar:
https://raw.githubusercontent.com/EclipseFdn/iot.eclipse.org/master/content/_index.md

This looks like a bug.

Reply:
the main homepage originally does not show sidebar. Is not that by design? 🤔

Reply:
Yes, I don't expect a sidebar on the homepage.

Reply:
So I'm not sure about the other sidebars, are they in use on any pages? 🤔 Either they are bug or abandoned but forgot to remove the code. I think we need to confirm that.

Reply:
FYI: on project and adopters page, they are also hide_sidebar: true.

Reply:
We are using them:
https://iot.eclipse.org/adopters/testbeds/propose-a-new-testbed/

I expect you to display that mobile dropdown if the page has sidebar links. It's outside of the scope of this bug for you to worry if a sidebar should be there or not.

We can address these 2 different issues separately.

Reply:
I am making the accordion as Martin suggested https://github.com/EclipseFdn/iot.eclipse.org/issues/350#issuecomment-674102182. I can make a dropdown too to compare. 

I think we need something more general that can be used for all those using sidebars.

Reply:
I am fine with either solution but I agree the goal here is to create something that we can use for all pages with a menus.sidebar

Reply:
I found them in a sub-page of adopters, sorry it was hard to notice for me, bringing them back!